src/PrestaShopBundle/EventListener/TokenizedUrlsListener.php

```diff
@@ -30,7 +30,7 @@
 use PrestaShop\PrestaShop\Adapter\LegacyContext;
 use PrestaShop\PrestaShop\Core\Feature\TokenInUrls;
 use Symfony\Component\HttpFoundation\RedirectResponse;
-use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
```

Comment:
PHP CS Fixer requests that KernelEvent is above RequestEvent not below

Reply:
Fixed